llama_index/embeddings/bedrock.py

```diff
@@ -0,0 +1,184 @@
+from typing import Any, List, Optional
+import warnings
+import json
+import os
+from llama_index.bridge.pydantic import PrivateAttr
+from llama_index.embeddings.base import BaseEmbedding
+
+Embedding = List[float]
+
+
+class Models(str):
+    TITAN_EMBEDDING = "amazon.titan-embed-text-v1"
+
+
+# To be used when AWS Bedrock comes with more embedding models
+# class ModelParams(str):
+# TITAN_EMBEDDING = {"inputText":""}
+
+
+class BedrockEmbeddings(BaseEmbedding):
```

Comment:
can we rename to `BedrockEmbedding` for naming consistency?

Reply:
Yes, definitely, will do it.